client/app/hosting/translations/Messages_fr_FR.xml

```diff
@@ -1520,4 +1520,19 @@
    <translation id="hosting_dashboard_general_informations" qtlid="27115">Informations générales</translation>
    <translation id="hosting_dashboard_configuration" qtlid="4406">Configuration</translation>
    <translation id="hosting_dashboard_access" qtlid="82449">Accès</translation>
+
+   <translation id="hosting_dashboard_subscription">Abonnement</translation>
+   <translation id="hosting_dashboard_subscription_contacts">Contacts</translation>
+   <translation id="hosting_dashboard_subscription_creation">Date de création</translation>
+   <translation id="hosting_dashboard_subscription_manual_renew">À renouveler le</translation>
+   <translation id="common_subscription_contacts_admin">Administrateur</translation>
+   <translation id="common_subscription_contacts_billing">Facturation</translation>
+   <translation id="common_subscription_contacts_tech">Technique</translation>
+
+   <translation id="expiration_renew_date_label">Renouvellement automatique prévu en </translation>
```

Comment:
You can trim trailing whitespace.

Furthermore these translations must be moved right next to the component.

Reply:
Nop, the date should be included in the translation, because there could be languages where the date is in the middle of the string: `Die automatische Verlängerung ist für {{renewalDate}} geplant`

Reply:
Is the translation handled in the date filter ?

Reply:
I noticed that you are targeting this kind of result:

![fix_web_hosting_dashboard](https://user-images.githubusercontent.com/428384/52616968-9a4ba200-2e9a-11e9-958c-fd248688d16f.png)

It sounds a bit difficult to put the date inside the translations at this point.

Do you have any suggestions @FredericEspiau?

It will probably helps us in upcoming changes.